ios/Classes/components/instant/InstantAdvancedComponent.swift

```diff
@@ -51,12 +51,14 @@ class InstantAdvancedComponent: BaseInstantComponent, InstantComponentProtocol {
             onError(paymentEventDTO: paymentEventDTO)
         case .action:
             onAction(paymentEventDTO: paymentEventDTO)
+        case .update:
+            return
```

Comment:
Some components are just silently return, Apple Pay has a dedicated comment for code clarity, just an observation.

Reply:
Good suggestion, will add a comment here as well to enhance code clarity.